skel/templates/etc/scripts/runTests.sh

```diff
@@ -153,22 +185,13 @@ if [ ${#INVALID_OPTIONS[@]} -ne 0 ]; then
     exit 1
 fi
 
-# Move "8.1" to "php81", the latter is the docker container name
+# Move "7.2" to "php72", the latter is the docker container name
 DOCKER_PHP_IMAGE=`echo "php${PHP_VERSION}" | sed -e 's/\.//'`
 
 # Set $1 to first mass argument, this is the optional test file or test directory to execute
 shift $((OPTIND - 1))
 if [ -n "${1}" ]; then
-    TEST_FILE="Web/typo3conf/ext/toujou_{{snake_case_projectname}}/${1}"
-else
-    case ${TEST_SUITE} in
-        unit)
-            TEST_FILE="Web/typo3conf/ext/toujou_{{snake_case_projectname}}/Tests/Unit"
-            ;;
-        functional)
-            TEST_FILE="Web/typo3conf/ext/toujou_{{snake_case_projectname}}/Tests/Functional"
-            ;;
-    esac
+    TEST_FILE="Web/typo3conf/ext/toujou_form_engine/${1}"
```

Comment:
should be `{{snake_case_projectname}}` i guess?

Reply:
@helsner yes. thanks